### ISSUE TITLE
Weird error when using /cluster/:node/logs

### ISSUE THREAD

**Issue by jesusgn90** [wazuh/wazuh-api#190]
- Cluster enabled + running -> very slow for worker nodes, timeout is low
- Cluster enabled + NOT running -> `{"error":3012,"message":"Cluster is not running."}`
- Cluster disabled -> `{"error":1000,"message":"ossec_log() got an unexpected keyword argument 'node_id'"}`
- Invalid node -> `{"error":3009,"message":"Error executing request to internal socket: No JSON object could be decoded"}`

**Comment by jesuslinares** [comment]
If the cluster is disabled, we should disable the /cluster/:node calls.

**Comment by mgmacias95** [comment]
Hello @jesusgn90 and @jesuslinares,

The issues described in this issue have been fixed in Wazuh v3.9.0:


### Cluster enabled + running -> very slow for worker nodes, timeout is low

As you can see in the following table, the execution time is almost the same no matter where the API call is executed:

| Node where the request is executed | Node where logs are being fetched | Execution time | Cluster disabled |
|:----------------------------------:|:---------------------------------:|:--------------:|:----------------:|
| Master                             | Master                            | 0.380s         | No               |
| Master                             | Worker                            | 0.389s         | No               |
| Worker                             | Master                            | 0.372s         | No               |
| Worker                             | Worker                            | 0.399s         | No               |

### Cluster enabled + NOT running

If the cluster is not running, an error is correctly returned:

```javascript
# curl -u foo:bar "localhost:55000/cluster/worker-1/logs?pretty"{
   "error": 3012,
   "message": "Cluster is not running"
}
```

### Cluster disabled

Fixed in https://github.com/wazuh/wazuh/pull/2597. Now the following error is shown:

```javascript
# curl -u foo:bar "localhost:55000/cluster/worker-1/logs?pretty"
{
   "error": 3013,
   "message": "Cluster is disabled"
}
```

### Invalid node

The following error is returned:

```javascript
# curl -u foo:bar "localhost:55000/cluster/worker-2/logs?pretty"
{
   "error": 3022,
   "message": "Unknown node ID"
}
```

Best regards,
Marta
